tests/performanceplatform/utils/test_collector_config.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from hamcrest import assert_that, equal_to
+from mock import patch
+from performanceplatform.utils.collector import get_config
+
+
+class ConfigTest(unittest.TestCase):
+    @patch('performanceplatform.client.collector.CollectorAPI.get_collector')
```

Comment:
The purpose of the test during development was to make sure that the data returned from stagecraft was being mapped into the format that pp-collector is expecting.

It can be removed.